src/BPEssentials/Commands/Chat/Sudo.cs

```diff
@@ -0,0 +1,15 @@
+﻿using BrokeProtocol.Entities;
+using BPEssentials.Utils;
+using BPEssentials.Abstractions;
+
+namespace BPEssentials.Commands
+{
+    public class Sudo : Command
+    {
+        public void Invoke(ShPlayer player, ShPlayer target, string message)
+        {
+            Core.Instance.Logger.LogInfo($"[GLOBAL] {target.username}: {message}");
```

Comment:
```suggestion
            Core.Instance.Logger.LogInfo($"[SUDO] {target.username}: {message}");
```
Maybe marking it as SUDO to prevent confusion